contrib/pkg/broker/user_provided/controller/controller_test.go

```diff
@@ -17,8 +17,12 @@ limitations under the License.
 package controller
 
 import (
+	"github.com/kubernetes-incubator/service-catalog/contrib/pkg/broker/controller"
 	"testing"
 )
 
+// Make sure that userProvidedController implements Controller interface
+var _ controller.Controller = &userProvidedController{}
```

Comment:
@MHBauer not blocking for this PR, but can you tell my offline why you like these?